modules/ROOT/pages/design-view-source-from-exchange.adoc

```diff
@@ -0,0 +1,24 @@
+= View the Source Project for Assets Published to Exchange
+
+After publishing an asset to Anypoint Exchange, you can view the source files in the API Designer. Exchange provides a direct link to the API Designer project containing the source files.
+
+== Procedure
+
+. Open the API specification or fragment in Anypoint Exchange.
+. Open the pop-up menu for the asset version whose source you want to view.
+. Select *View code in API Designer*
```

Comment:
View Code is also available from the toolbar option on top in Exchange page, this opens latest publish version 
![image](https://user-images.githubusercontent.com/22245749/96884015-6c807400-1457-11eb-8ed5-99498c583613.png)

Reply:
Updated the content.

Reply:
Line 9 says "in API Designer" but the image doesn't.